core/tests/protractor/accessibility.js

```diff
@@ -29,28 +29,30 @@ describe('screenreader and keyboard user accessibility features', function() {
     libraryPage = new LibraryPage.LibraryPage();
   });
 
-  it('should skip to the main content element', function() {
-    libraryPage.get();
-    browser.actions().sendKeys(protractor.Key.TAB).perform();
+  it('should skip to the main content element', async function() {
+    await libraryPage.get();
+    await browser.actions().sendKeys(protractor.Key.TAB).perform();
     var skipLink = element(by.css('.protractor-test-skip-link'));
-    waitFor.elementToBeClickable(skipLink, 'Could not click skip link');
-    skipLink.click();
+    await waitFor.elementToBeClickable(skipLink, 'Could not click skip link');
+    await skipLink.click();
     var mainContent = element(by.css('.protractor-test-main-content'));
-    expect(mainContent.getAttribute('id'))
-      .toEqual(browser.driver.switchTo().activeElement().getAttribute('id'));
+    expect(await mainContent.getAttribute('id')).toEqual(
+      await (await browser.driver.switchTo().activeElement())
```

Comment:
Do we need the double `await` here? We do it differently on line 157 of core/tests/protractor_utils/CollectionEditorPage.js

Reply:
Yeah, I tried logging `browser.driver.switchTo().activeElement()` and found it to be a WebElementPromise object, so I added await for it.

Reply:
Hmmm, this looks weird actually.... wouldn't this actually result in 
```suggestion
      await await browser.driver.switchTo().activeElement()
```
which might be the same as the single await right?

Reply:
@nithusha21 I think you're missing the `.getAttribute('id')` on the next line. The first `await` is for the `.getAttribute('id')`, while the second `await` is for the `.activeElement()`

Reply:
I'm resolving this since the double await is indeed required